proposed/0002-release-schedule.md

```diff
@@ -0,0 +1,87 @@
+until a MAJOR feature is released so 21.0.0 and so forth. If there is then a security patch it would be released for the latest version
+(e.g. 21.0.1) and only backported to 20.2.1 (via a branch called v20) and 19.4.22 (via a branch called v19) in this case
+if those branches were less than two years old at the time.
+
+#### Effects
+
+- Relegates v19 to security-only going forward and it basically becomes EOL in 2 years.
+- All new releases are at least minor versions so it is understood there is basically always a small upgrade risk. This resolves 
+  the common issue where a feature is BC breaking but in a very small way - this is always acceptable.
+- The "which branch?" debate is resolved because all PRs go to "main" which is also indefinitely the default checkout branch.
+- Bleeding-edge users and can easily use "dev-main" indefinitely.
+- Active users can upgrade as new releases are announced using the major/minor numbers as guidance.
+- Users wanting minimal upgrade hassle always have at least 2 years to get up to date to the latest major version for another two years or more.
+
+### Alternative 2
+
+We will follow the example of Ubuntu, which bases the Major Version Number on the Year.  
+And also has a specific Rythmus of an LTS version every 4 Releases,
+adding bigger experimantal features in the Major Release after an LTS version.
+
+But we will go with just one Major Release each year, which results in one LTS Release every 4 years.
+
+
+#### Visualization of the Ubuntu Release cycle
+
+![ubuntu release stragey visualized](../assets/Ubuntu_release_cycle_Ubuntu.png?raw=true "ubuntu release stragey visualized")
+
+### Alternative 3.1
+
+Have one Minor Release each month, do Major Releases every time a BC break is part of the Release. Only one currently supported Branch.
+
+### Alternative 3.2
+
+Have one Minor Release every Quarter
+
+
+## Implementation
+
+{{Give a high-level overview of implementation requirements and concerns. Be specific about areas of code that need to change, and what their potential effects are. Discuss which repositories and sub-components will be affected, and what its overall code effect might be.}}
+
+{{THIS SECTION IS REQUIRED FOR RATIFICATION -- you can skip it if you don't know the technical details when first submitting the proposal, but it must be there before it's accepted}}
+
+## Prior Art
+
+
+{{This section is optional if there are no actual prior examples in other tools}}
+
+{{Discuss existing examples of this change in other tools, and how they've addressed various concerns discussed above, and what the effect of those decisions has been}}
+
+## Unresolved Questions and Bikeshedding
+
+{{Write about any arbitrary decisions that need to be made (syntax, colors, formatting, minor UX decisions), and any questions for the proposal that have not been answered.}}
+
+{{THIS SECTION SHOULD BE REMOVED BEFORE RATIFICATION}}
```

Comment:
these sections should be removed, they're not filled

Reply:
Updated although Alternative 1 and 2 leave some unresolved questions..